exchange/bitswap/workers.go

```diff
@@ -9,16 +9,20 @@ import (
 	u "github.com/ipfs/go-ipfs/util"
 )
 
+var TaskWorkerCount = 4
```

Comment:
we can also try 16/32 and see how it goes. if not good we can bring it back down.